packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -232,17 +232,21 @@ export class CucumberJSAllureFormatter extends Formatter {
     const labels = this.parseTagsLabels(scenario?.tags || []);
     const links = this.parseTagsLinks(scenario?.tags || []);
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
+    const thread = ALLURE_THREAD_NAME || process?.getuid()?.toString();
```

Comment:
maybe  we can use `process.pid` for compatibility

Reply:
Why not, good catch